src/annotator/events.ts

```diff
@@ -0,0 +1,16 @@
+import type { SidebarLayout } from '../types/annotator';
+
+type LayoutChangeEventDetail = {
+  sideBySideActive: boolean;
+  sidebarLayout: SidebarLayout;
+};
+
+export class LayoutChangeEvent extends CustomEvent<LayoutChangeEventDetail> {
+  constructor(detail: LayoutChangeEventDetail) {
+    super('hypothesis:layoutchange', {
+      bubbles: true,
+      cancelable: false,
```

Comment:
A consideration for the future: do we allow canceling this event via `event.preventDefault()`? I can't see a legitimate use case at the moment, but maybe one will appear.